README.md

```diff
@@ -33,6 +33,7 @@ RpiAuth.configure do |config|
   config.identity_url = 'http://localhost:3002'        # The url for the profile instance being used for auth
   config.user_model = 'User'                           # The name of the user model in the host app being used, use the name as a string, not the model itself
   config.scope = 'openid email profile force-consent'  # The required OIDC scopes
+  config.session_keys_to_persist = 'foo bar'           # Optional: any session keys to persist across sessions (as the session is reset upon log in)
```

Comment:
Does this need to be a string or could we use an array here, that would avoid the need for the `split` when processing it?

Reply:
Yes good point, my thinking behind string was:

* It matches the input format used elsewhere, such as `config.scope`
* If taking config values from env vars, we should assume string (though, I'm not convinced we would usually want to utilise env vars for `config.session_keys_to_persist` values, more likely it's going to set in the initializer and will be same for all envs)

Reply:
I see that the test uses an array here so would be good to have the example and the test consistent too?

Reply:
Good spot, yep fixing...